sw/device/lib/dif/dif_otbn.h

```diff
@@ -61,41 +61,35 @@ typedef enum dif_otbn_result {
    * This error is recoverable and the operation can be retried after correcting
    * the problem with the argument(s).
    */
-  kDifOtbnBadArg,
-
-  /**
-   * The hardware returned unexpected data.
-   *
-   * This error either indicates malfunctioning hardware, or a mismatch between
-   * software and hardware.
-   */
-  kDifOtbnUnexpectedData,
+  kDifOtbnBadArg
 } dif_otbn_result_t;
 
 /**
- * Error codes received from the hardware.
+ * OTBN Errors
+ *
+ * OTBN uses a bitfield to indicate which errors have been seen. Multiple errors
+ * can be seen at the same time. This enum gives the individual bits that may be
+ * set for different errors.
  */
-typedef enum dif_otbn_err_code {
-  // Keep error codes in sync with the hardware!
-  /** No error occurred. */
-  kDifOtbnErrCodeNoError = 0x0,
+typedef enum dif_otbn_err_bit {
+  kDifOtbnErrBitNoError = 0,
   /** Load or store to invalid address. */
-  kDifOtbnErrCodeBadDataAddr = 0x1,
+  kDifOtbnErrBitBadDataAddr = (1 << 0),
```

Comment:
```suggestion
  kDifOtbnErrBitBadDataAddr = (1 << OTBN_ERR_CODE_BAD_DATA_ADDR_BIT),
```

(See also the comment in dif_otbn.c regarding the static asserts.)